proto/tendermint/liquidity/v1beta1/liquidity.proto

```diff
@@ -1,18 +1,21 @@
 syntax = "proto3";
-package tendermint.liquidity;
+package tendermint.liquidity.v1beta1;
 
-import "tx.proto";
+import "tendermint/liquidity/v1beta1/tx.proto";
 import "gogoproto/gogo.proto";
 import "cosmos_proto/coin.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 
 option go_package = "github.com/tendermint/liquidity/x/liquidity/types";
 option (gogoproto.goproto_getters_all) = false;
 
+// Structure for the pool type to distinguish the characteristics of the reserve pools
 message PoolType {
     option (gogoproto.equal) = true;
 
-    // id of target pool type, only 1 is allowed on this version.
+    // This is the id of the pool_type that will be used as pool_type_id for pool creation.
+    // In this version, pool-type-id 1 is only available
+    // {"id":1,"name":"ConstantProductLiquidityPool","min_reserve_coin_num":2,"max_reserve_coin_num":2,"description":""}
```

Comment:
the best guidance is not to mention "for this version" since each version is what it is.
- we don't mention future features (for example, if we say "for this version" it implies that future versions will support other pool type id) see https://developers.google.com/style/future 
- we always use present tense, see https://developers.google.com/style/tense

Reply:
for line 16 above, do we mean "Defines the pool_type_id of the created pool." ?